README.md

```diff
@@ -95,3 +99,12 @@ and [Sprig](https://github.com/Masterminds/sprig) library.
 ### Built-in functions
 There are a few built in functions as well:
   * `require (env "ENV_NAME")` - Renders an error if environments variable does not exists. If it is equal to empty string, returns empty string.  `{{ require (env "TIMEOUT_MS) }}`
+
+### Nested GO templates
+If you have nested GO templates there is problem with escaping. To resolve this problem you can define different 
+delimiters of template tags using `--delim-left` and `--delim-right` command line parameters.
```

Comment:
`--delim-left` => `-delim-left`, same for right

Reply:
fixed in https://github.com/seznam/goenvtemplator/pull/14/commits/ad360b91d58314104e6512d8d6b70b159ab790ad

but works both way